test/browser_tests/spec_suites/shared/shared_footer.js

```diff
@@ -8,6 +8,8 @@ describe( 'The Footer Component', function() {
   var _footerLinkLookup =
   { '/offices/accessibility/':
       'accessibility',
+    '/offices/office-of-civil-rights/':
+      'Office of Civil Rights',
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/71920e9c47d19820f9b6a2daa60d56addc28aa1b